Sources/AnyEncodable.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  AnyEncodable.swift
+//  AppStoreConnect-Swift-SDK
+//
+//  Created by Rui Costa on 14/03/2020.
+//
+
+import Foundation
+
+public struct AnyEncodable: Encodable {
```

Comment:
Can we get around not making this a public struct?

Reply:
Oops of course!